pkg/render/manager.go

```diff
@@ -153,7 +157,8 @@ type ManagerConfiguration struct {
 	ComplianceLicenseActive bool
 
 	// Whether the cluster supports pod security policies.
-	UsePSP bool
+	UsePSP    bool
+	Namespace string
```

Comment:
We need to pass the namespace in to the render component now instead of hard-coding it.

Would be good to _force_ this behavior - e.g., delete the `render.ManagerNamespace` variable. This will ensure every component is doing the right thing.